pages/documentation.html

```diff
@@ -106,6 +106,11 @@
 with the SSReflect proof language, and the Mathematical Components
 library.</li>
 
+<li><a href="">Hydras &amp; Co.</a> (Pierre Castéran <em>et al.</em>, 2021),
```

Comment:
I think the following suggestion could be a way of insisting on the continuously in-progress aspect.

```suggestion
<li><a href="">Hydras &amp; Co.</a> (Pierre Castéran <em>et al.</em>, 2021–⁠),
```

Reply:
How about "2021-ongoing"?

Reply:
Note that I was using an en-dash. Your call on the "ongoing" addition, but that's not something that I've seen anywhere before.

Reply:
"2021–⁠present" then?

Reply:
Sounds better to me.